Yosemite/YosemiteTests/SnapshotsProvider/FetchResultSnapshotsProviderTests.swift

```diff
@@ -0,0 +1,239 @@
+import XCTest
+import TestKit
+import CoreData
+import Combine
+
+@testable import Yosemite
+import Storage
+
+@available(iOS 13.0, *)
+final class FetchResultSnapshotsProviderTests: XCTestCase {
+
+    private var storageManager: MockupStorageManager!
+
+    private var cancellables = Set<AnyCancellable>()
+
+    var viewStorage: StorageType {
```

Comment:
nit: could this be private?
```suggestion
    private var viewStorage: StorageType {
```

Reply:
I don't know why I keep missing `private` in computed properties. This doesn't feel like the first time I did this. 😄 Thanks for catching it! Fixed in 0e617e4903d9194b0fd8d5e1cf3518a602d05049.